src/exporters/vhi.py

```diff
@@ -0,0 +1,220 @@
+"""
+functionality:
+1) download VHI data in parallel
+2)
+
+CDS functions:
```

Comment:
Nit: there are lots of comments in this file which could be removed.

Reply:
think i've removed alot of them